translations/messages-en.properties

```diff
@@ -274,7 +274,7 @@ Error\ saving\ feedback = Error saving feedback
 feedback.submitted = Feedback submitted
 Error\ updating\ group = Error updating group
 Antenatal\ Care = Antenatal Care
-Stock\ Monitoring = Stock Monitoring
+Reporting\ Rates = Reporting Rates
```

Comment:
All the translations will say "Stock Monitoring" in whatever language, which would be incorrect.  So in this case, I think renaming keys is entirely correct.